internal/controllers/bundledeployment/bundledeployment.go

```diff
@@ -151,15 +174,23 @@ type controller struct {
 	storage          storage.Storage
 	releaseNamespace string
 
+	unpacker          unpackersource.Unpacker
 	controller        crcontroller.Controller
+	finalizers        crfinalizer.Finalizers
 	dynamicWatchMutex sync.RWMutex
 	dynamicWatchGVKs  map[schema.GroupVersionKind]struct{}
 }
 
+//+kubebuilder:rbac:groups=core.rukpak.io,resources=bundledeployments/finalizers,verbs=update
 //+kubebuilder:rbac:groups=core.rukpak.io,resources=bundledeployments,verbs=list;watch
 //+kubebuilder:rbac:groups=core.rukpak.io,resources=bundledeployments/status,verbs=update;patch
-//+kubebuilder:rbac:groups=core.rukpak.io,resources=bundledeployments/finalizers,verbs=update
+//+kubebuilder:rbac:verbs=get,urls=/bundles/*;/uploads/*
+//+kubebuilder:rbac:groups=core,resources=pods,verbs=list;watch;create;delete
+//+kubebuilder:rbac:groups=core,resources=configmaps,verbs=list;watch
+//+kubebuilder:rbac:groups=core,resources=pods/log,verbs=get
 //+kubebuilder:rbac:groups=*,resources=*,verbs=*
```

Comment:
😄 (I realize this wasn't a change from this PR)

Reply:
Do you mean: `//+kubebuilder:rbac:groups=*,resources=*,verbs=*`?

I guess this was to be able to handle any bundle objects. Not sure if we need wildcard for verbs, but can create an issue to discuss more on handling this permission scoping.

Reply:
https://github.com/operator-framework/rukpak/issues/800 for follow up.